packages/docs/stories/foundations/Tokens/Colors.mdx

```diff
@@ -0,0 +1,58 @@
+import { Meta } from "@storybook/blocks";
+import * as tokens from '@sick-design-system/design-tokens';
+
+<Meta title="Foundations/Design Tokens/Colors" />
+
+# Colors
+
+<div className="color-swatches">
+  {
+    Object
+      .entries(tokens)
+      .filter(([token]) => token.toLowerCase().startsWith('sdscolor'))
+      .map(([token, value]) => (
+        <div
+          className="color-swatch-wrapper"
+          title={token}
+          key={token}
+        >
+          <div className="color-swatch" style={{ backgroundColor: value }} />
+          <div className="color-swatch-label">
+            {token}
```

Comment:
It depends on which "token type" you use, if the naming of the colors is correct. Like this the token name is correct for the **js** tokens. But would be incorrect for the **css** or **scss**.
Should we address all names? 
Instead of:
![image](https://github.com/SickDesignSystem/sds/assets/115139104/78968290-091c-473e-bd96-ca932ba70f21)

we could do following:
![colors](https://github.com/SickDesignSystem/sds/assets/115139104/863c0379-4e54-4562-9a7d-1b50da83d95f)

Reply:
Excellent suggestion. I will have a look into this :)

Reply:
After talking to stakeholders, I have adjusted the table once more:

1. Tokens are now grouped by color group (Primary, Secondary, Accent, ...)
2. You are now able to copy all token values for the corresponding actions via clicking on the table cell or the Save Icon.

![table-2](https://github.com/SickDesignSystem/sds/assets/141909690/db8bb3e0-abf3-49e8-8fe2-54ac17824aee)